proxysql/README.md

```diff
@@ -14,6 +14,18 @@ The ProxySQL integration is included in the [Datadog Agent][3] package, so you d
 
 ### Configuration
 
+#### Enabling SSL
```

Comment:
Add in here about pymysql/proxysql auto-guess, and don't need to have `use_tls`.